packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -57,8 +59,9 @@ export class Checkboxes extends GOVUKFrontendComponent {
 
       // Throw if target conditional element does not exist.
       if (!document.getElementById(targetId)) {
-        throw new ElementError(`#${targetId}`, {
-          componentName: 'Checkboxes'
+        throw new ElementError({
+          componentName: 'Checkboxes',
+          identifier: `Conditional element #${targetId}`
```

Comment:
I'm torn on "element" creeping back in, but does everyone call these "Conditional reveals"?

```suggestion
          identifier: `Conditional reveal (\`#${targetId}\`)`
```

Reply:
Yeah, let's go for the wider use rather than trying to shimmy everything into a certain format.

Reply:
Brill. Then there's the HTML suggestion from below too:

```suggestion
          identifier: `Conditional reveal (\`id="${targetId}"\`)`
```